packages/components/src/navigation/NavTab.tsx

```diff
@@ -67,17 +67,21 @@ const NavTab = memo(
               data-testid={`btn-nav-tab-${title}`}
               role="tab"
               tabIndex={0}
-              onAuxClick={event => {
-                // e.button equaling 1 corresponds to middle mouse button being clicked
-                if (isClosable && event.button === 1) {
+              onAuxClick={e => {
+                // button equaling 1 corresponds to middle mouse button being clicked, and buttons equaling 0 corresponds to it being the only button pressed
+                if (isClosable && e.button === 1 && e.buttons === 0) {
                   onClose?.(key);
                 }
               }}
               onClick={e => {
+                // have to have seperate check onClick for Safari not supporting AuxClick
+                if (isClosable && e.button === 1 && e.buttons === 0) {
```

Comment:
I don't think `onClick` this will fire in Safari for middle mouse button right?

Reply:
Seems to be working

Reply:
I guess `onClick` behaves differently than the jQuery `click` where I tested in `Tab`

Reply:
Yeah, this behaviour is definitely weird, but there was something about Safari will call regular click for a middle click, so I'm pretty sure that's why this works.

Reply:
Yeah, 'click' fires in Safari for me now. I thought it was not when I tested earlier. Looks good.